.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+name: docker
+on: push
+jobs:
+  docker:
+    runs-on: ubuntu-20.04
+    if: github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/tags')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
I might be missing something important, but why do you need qemu?

Reply:
Ah yes, I was in need of this for some other project, so here we probably don't need it and I removed it.